static/css/partials/_partials.scss

```diff
@@ -10,6 +10,7 @@
 @import "buttons";
 @import "large-icon-block";
 @import "share-panel";
+@import "signin-fxA";
```

Comment:
The uppercase `A` here is inconsistent with the rest of the filenames.

Reply:
Renamed to use lowercase 'a'